### PR TITLE
pre-collections: small issues fixed

### PR DESCRIPTION
**Pull request by barpavel**
Solving a few issues to enhance moving to collections:

1. Removing unused path variable in "generate_vars.py" containing the "disaster-recovery" substring (that otherwise should have been changed in the collections project).
2. Fixing a bug with user input in "validator.py".
3. A few improvements in "validator.py"

Signed-off-by: Pavel Bar <pbar@redhat.com>

### CODE REVIEW

**Comment by barpavel** [comment]
Verified today, 14/10/2020.

File: files/validator.py [hunk 1]
```diff
@@ -425,24 +416,25 @@ def _validate_duplicate_keys(self, var_file):
...
         domains = 'domains'
         roles = 'roles'
-        aff_group = 'aff_groups'
-        aff_label = 'aff_labels'
+        aff_groups = 'aff_groups'
+        aff_labels = 'aff_labels'
         network = 'network'
         key1 = 'primary_name'
         key2 = 'secondary_name'
+        dr_primary_name = 'dr_primary_name'
```
Comment (shenitzky) [inline files/validator.py:421]: used only once, why do we need it as a variable?

**Review by shenitzky — CHANGES_REQUESTED** [review]

File: files/validator.py (hunk 1 above)
Comment (barpavel) [reply files/validator.py:421]: > used only once, why do we need it as a variable?

1) For consistency with others (although I agree that the existing variables are used at least twice).
2) To be able to fit "[domains, self.domain_map, dr_primary_name, dr_secondary_name]," into 1 line.

But that's not critical, I can revert this part.
I thought that it improves the readability, especially that this method already defines 8 similar string variables, so 2 new ones match the style and don't look ugly :)

Comment (shenitzky) [reply files/validator.py:421]: Not that critical, leave it as is

**Review by shenitzky — APPROVED** [review]

**Pull request merged**